CRM/Core/BAO/Navigation.php

```diff
@@ -928,13 +928,22 @@ public static function buildHomeMenu(&$menu) {
           ];
         }
         else {
+          $item['child'][] = [
+            'attributes' => [
+              'label' => ts('My Account'),
+              'name' => 'My Account',
+              'url' => 'civicrm/my-account',
```

Comment:
I think it might be good to use "user" rather than "account" to keep with existing terminology. I think account might also mean other things in other contexts - here is a User Account but you can have Bank Accounts, maybe Membership Accounts(?)

Reply:
E.g. maybe `civicrm/user/me` would keep things bundled together with `civicrm/user/edit`

Reply:
It's a bit unfortunate that the contact dashboard appears at `civicrm/user`...

Reply:
I changed it to "My User Account". "My User" just sounds weird - it's not like there's a separate person in there somewhere. I also changed it to `civicrm/user/me`.